src/ray/common/test_util.cc

```diff
@@ -138,4 +243,17 @@ std::string REDIS_MODULE_LIBRARY_PATH;
 /// Ports of redis server.
 std::vector<int> REDIS_SERVER_PORTS;
 
+/// Path to object store executable binary.
+std::string STORE_EXEC_PATH;
+
+/// Path to gcs server executable binary.
+std::string GCS_SERVER_EXEC_PATH;
+
+/// Path to raylet executable binary.
+std::string RAYLET_EXEC_PATH;
+/// Path to mock worker executable binary. Required by raylet.
+std::string MOCK_WORKER_EXEC_PATH;
+/// Path to raylet monitor executable binary.
+std::string RAYLET_MONITOR_EXEC_PATH;
```

Comment:
maybe better to add  a TEST prefix for all these constants above, or put this file into ray::test namespace instead of ray?

Reply:
ray::test is better since there're other variables and functions in this namespace that are also used for tests.

Reply:
after second thoughts, adding test namespace will make it inconsistent with all other test cases which have no test namespace included. 
TEST prefix added for simplification.